visualization/app/codeCharta/ui/copyToClipboardButton/util/clipboardString.ts

```diff
@@ -0,0 +1,21 @@
+import { FileToValue } from "./getFilenamesWithHighestMetrics"
+
+function getStringLineFromItem(item: FileToValue): string {
+	return `\t${String.fromCodePoint(8226)} ${item.name} (${item.value})` + `\n`
```

Comment:
can we use the real code symbol directly?

Reply:
Yes! Resolved with commit [89ab875](https://github.com/MaibornWolff/codecharta/pull/2942/commits/89ab875127f1d597acf125a17d0178e19e5096b9)